src/CountryPicker.js

```diff
@@ -275,21 +279,22 @@ export default class CountryPicker extends Component {
       position = this.listHeight - this.visibleListHeight
     }
 
-    // scroll
-    this._listView.scrollTo({
-      y: position
-    })
+<<<<<<< HEAD
```

Comment:
It seems you left this an unresolved conflict.

Reply:
Right. What is the proper way to write this so Lint is happy?

Reply:
I'd say `this._flatList.scrollToIndex({ index })` is the best of both.